test/cbmc/proofs/Makefile-jobs.common

```diff
@@ -0,0 +1,33 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY=harness
+
+ifdef CBMC_MAX_BUFSIZE
+    DEFINES += -DCBMC_MAX_BUFSIZE=$(CBMC_MAX_BUFSIZE)
+endif
+ifdef THINGNAME_MAX_LENGTH
+    DEFINES += -DTHINGNAME_MAX_LENGTH=$(THINGNAME_MAX_LENGTH)
+endif
+ifdef JOBID_MAX_LENGTH
+    DEFINES += -DJOBID_MAX_LENGTH=$(JOBID_MAX_LENGTH)
+endif
+
+INCLUDES += -I$(CBMC_ROOT)/include
+INCLUDES += -I$(SRCDIR)/include
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROOF_SOURCES += $(PROOF_SOURCE)/mallocCanFail.c
+PROJECT_SOURCES += $(PROOFDIR)/jobs.c
+
+CHECKFLAGS += --pointer-primitive-check
+
+include ../Makefile.common
+
+# remove static constraints to facilitate proofs
+# replace some functions with stubs by renaming originals (see SED_EXPR in proof Makefiles)
+$(PROOFDIR)/jobs.c: $(SRCDIR)/jobs.c
+	sed -E '1s/^/#include "jobs_annex.h" /; s/^static //; $(SED_EXPR)' $< > $@
```

Comment:
I haven't found any documentation about the how to use this feature.  Even if it does let me access a static function, it seems like it would not solve how to replace a static function with a stub, given that all the statics live in a single source file.

The library has no static data, and having such would violate the tenets.  Knowing this, I used the simplest sed expression possible.  I don't think there's any need to future-proof this.